src/ts/Offers/api.ts

```diff
@@ -1,7 +1,7 @@
 import API from '../API';
 
 export function get() {
-  return fetch(API.offer, {
+  return window['fetch'](API.offer, {
```

Comment:
can't really use `window` as it will not be available on the server with server-side rendering. We could mock it tho